cluster/gce/gci/configure-helper-common.sh

```diff
@@ -2383,6 +2390,8 @@ function start-kube-controller-manager {
   sed -i -e "s@{{additional_cloud_config_volume}}@@g" "${src_file}"
   sed -i -e "s@{{pv_recycler_mount}}@${PV_RECYCLER_MOUNT}@g" "${src_file}"
   sed -i -e "s@{{pv_recycler_volume}}@${PV_RECYCLER_VOLUME}@g" "${src_file}"
+  sed -i -e "s@{{default_network_template_path_mount}}@${DEFAULT_NETWORK_TEMPLATE_PATH_MOUNT}@g" "${src_file}"
+  sed -i -e "s@{{default_network_template_path_volume}}@${DEFAULT_NETWORK_TEMPLATE_PATH_VOLUME}@g" "${src_file}"
```

Comment:
kube-controller-manager failed to start with this error in kubelet log:
```
Jan 26 02:27:08 sonyaperf1-012522-master kubelet[2565]: E0126 02:27:08.111613    2565 file.go:187] Can't process manifest file "/etc/kubernetes/manifests/kube-controller-manager.manifest": invalid pod: [spec.volumes[2].name: Invalid value: "default_network_template_path_mount": a DNS-1123 label must consist of lower case alphanumeric characters or '-', and must start and end with an alphanumeric character (e.g. 'my-name',  or '123-abc', regex used for validation is '[a-z0-9]([-a-z0-9]*[a-z0-9])?') spec.containers[0].volumeMounts[2].name: Not found: "default_network_template_path_mount"]
```

Reply:
The names of volume and mount are changed to "defaulttemplate" from "default_network_template_path_mount" in line 3397 and line 3398. 

In GCloud kube-up test environment, the changes have been tested. When publish, the changes are missed out.

Reply:
The new commit will be coming in 10 minutes.